package.json

```diff
@@ -81,8 +84,14 @@
     "debug": "^4.3.2",
     "enquirer": "^2.3.6",
     "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.6",
```

Comment:
Not needed. We already have `@wojtekmaj/enzyme-adapter-react-17` which is for react 17